test/DebugInfo/mandatoryinlining-wrongdebugscope.swift

```diff
@@ -0,0 +1,22 @@
+// RUN: %target-swift-frontend -emit-sil %s -Onone -Xllvm \
```

Comment:
I was just about to ask if I can remove this test because it's not really testing anything... but it looks like @gottesmm beat me to it :))